lighthouse-cli/test/smokehouse/test-definitions/perf/expectations.js

```diff
@@ -156,6 +156,67 @@ module.exports = [
     },
   },
   {
+    artifacts: {
+      TraceElements: [
+        {
+          'traceEventType': 'largest-contentful-paint',
+          'selector': 'body > div#late-content > img',
+          'nodeLabel': 'img',
+          'snippet': '<img src="../dobetterweb/lighthouse-480x318.jpg">',
+          'boundingRect': {
+            'top': 108,
+            'bottom': 426,
+            'left': 8,
+            'right': 488,
+            'width': 480,
+            'height': 318,
+          },
+        },
+        {
+          'traceEventType': 'cumulative-layout-shift',
+          'selector': 'body > h1',
+          'nodeLabel': 'Please don\'t move me',
+          'snippet': '<h1>',
+          'boundingRect': {
+            'top': 465,
+            'bottom': 502,
+            'left': 8,
+            'right': 352,
+            'width': 344,
+            'height': 37,
+          },
+          'score': 0.05873307291666667,
+        },
+        {
+          'traceEventType': 'cumulative-layout-shift',
+          'selector': 'body > div#late-content > span',
+          'nodeLabel': 'Sorry!',
+          'snippet': '<span>',
+          'boundingRect': {
+            'top': 426,
+            'bottom': 444,
```

Comment:
Is `boundingRect.bottom` flaky?  The smoke tests keep failing on this.

Reply:
could replace this with
```suggestion
            'bottom': '444 +/- 1',
```
 
and same for height?

Reply:
Should this apply to every trace element or just the ones which are confirmed to be flaky?

Reply:
I would only do it to ones that seem flaky. I'm guessing this is failing because of OS-specific font line-height rendering

Reply:
> Should this apply to every trace element or just the ones which are confirmed to be flaky?

ideally it should be as few as are needed (or OTOH if it's a value that's known to vary by design, then it makes sense to use bounds for all of them).

`boundingRect` should be deterministic if the test page is deterministic. Is there a reason this element is flaky and we could fix in the test fixture instead? If not, maybe leave a comment about why it might vary so the next person editing this file doesn't just think we were lazy in accepting a flaky test? :)

Reply:
I was too slow :)

> I'm guessing this is failing because of OS-specific font line-height rendering, could alternatively set a fixed height on the element instead

since there's nothing testing OS-specific font rendering, this fix makes sense to me to tighten up the testing we do care about